src/main/kotlin/no/nav/syfo/infrastructure/database/repository/OppfolgingsoppgaveRepository.kt

```diff
@@ -168,6 +171,29 @@ private fun DatabaseInterface.getOppfolgingsoppgave(uuid: UUID): POppfolgingsopp
     }
 }
 
+private const val queryGetActiveOppfolgingsoppgaverByPersonident = """
+    SELECT *
+    FROM HUSKELAPP
+    WHERE personident = ? AND is_active = true
+"""
+
+private fun DatabaseInterface.getActiveOppfolgingsoppgaver(personidenter: List<PersonIdent>): List<POppfolgingsoppgave> {
+    return if (personidenter.isEmpty()) {
+        emptyList()
+    } else {
+        connection.use { connection ->
+            connection.prepareStatement(queryGetActiveOppfolgingsoppgaverByPersonident).use {
+                val oppfolgingsoppgaver = mutableListOf<POppfolgingsoppgave>()
+                personidenter.forEach { personident ->
+                    it.setString(1, personident.value)
+                    oppfolgingsoppgaver.addAll(it.executeQuery().toList { toPOppfolgingsoppgave() })
+                }
+                oppfolgingsoppgaver
```

Comment:
Slipper instantiering hvis man bruker `map` og `flatten()` 😊
```suggestion
        connection.use { connection ->
            connection.prepareStatement(queryGetActiveOppfolgingsoppgaverByPersonident).use {
                personidenter.map { personident ->
                    it.setString(1, personident.value)
                    it.executeQuery().toList { toPOppfolgingsoppgave() }
                }.flatten()
            }
        }
```